src/algebra/module/localized_module.lean

```diff
@@ -387,4 +387,108 @@ instance localized_module_is_localized_module :
+    { dsimp, erw [mul_comm, f.map_smul, mul_smul], refl, },
+  end,
+  map_smul' := λ r x,
+  begin
+    induction x using localized_module.induction_on with a b,
+    erw [localized_module.lift_on_mk, localized_module.lift_on_mk, one_mul],
+    generalize_proofs h1,
+    dsimp,
+    erw [f.map_smul, h1.unit⁻¹.1.map_smul],
+    congr,
+  end }
+
+lemma from_localized_module.inj : function.injective $ from_localized_module S f :=
+λ x y eq1,
+begin
+  induction x using localized_module.induction_on with a b,
+  induction y using localized_module.induction_on with a' b',
+  simp only [from_localized_module, linear_map.coe_mk, localized_module.lift_on_mk] at eq1,
+  generalize_proofs h1 h2 at eq1,
+  erw [module.End_algebra_map_is_unit_inv_apply_eq_iff, ←linear_map.map_smul,
+    module.End_algebra_map_is_unit_inv_apply_eq_iff'] at eq1,
+  erw [localized_module.mk_eq, ←is_localized_module.eq_iff_exists S f, f.map_smul, f.map_smul, eq1],
+  refl,
+end
+
+lemma from_localized_module.surj : function.surjective $ from_localized_module S f :=
+λ x, let ⟨⟨m, s⟩, eq1⟩ := is_localized_module.surj S f x in ⟨localized_module.mk m s,
+by { rw [from_localized_module, linear_map.coe_mk, localized_module.lift_on_mk,
+    module.End_algebra_map_is_unit_inv_apply_eq_iff, ←eq1], refl }⟩
+
+lemma from_localized_module.bij : function.bijective $ from_localized_module S f :=
+⟨from_localized_module.inj _ _, from_localized_module.surj _ _⟩
+
+/--
+If `(M', f : M ⟶ M')` satisfies universal property of localized module, then `M'` is isomorphic to
+`localized_module S M` as an `R`-module.
+-/
+@[simps] noncomputable def iso : localized_module S M ≃ₗ[R] M' :=
+{ ..from_localized_module S f,
+  ..equiv.of_bijective (from_localized_module S f) $ from_localized_module.bij _ _}
+
+lemma iso_apply_mk (m : M) (s : S) :
+  iso S f (localized_module.mk m s) = (is_localized_module.map_units f s).unit⁻¹ (f m) :=
+rfl
+
+private lemma iso_symm_apply_aux (m : M') :
+  (iso S f).symm m = localized_module.mk (is_localized_module.surj S f m).some.1
+    (is_localized_module.surj S f m).some.2 :=
+begin
+  generalize_proofs _ h2,
+  apply_fun (iso S f) using linear_equiv.injective _,
+  rw [linear_equiv.apply_symm_apply],
+  simp only [iso_apply, linear_map.to_fun_eq_coe, from_localized_module_apply],
+  erw [localized_module.lift_on_mk, module.End_algebra_map_is_unit_inv_apply_eq_iff', h2.some_spec],
+end
+
+lemma iso_symm_apply' (m : M') (a : M) (b : S) (eq1 : b • m = f a) :
+  (iso S f).symm m = localized_module.mk a b :=
+(iso_symm_apply_aux S f m).trans $ localized_module.mk_eq.mpr $
+begin
+  generalize_proofs h1,
+  erw [←is_localized_module.eq_iff_exists S f, f.map_smul, f.map_smul, ←h1.some_spec, ←mul_smul,
+    mul_comm, mul_smul, eq1],
+end
+
+/--
+If `(M', f)` and `(M'', g)` both satisfy universal property of localized module, then `M', M''`
+are isomorphic as `R`-module
+-/
+noncomputable def unique_up_to_iso [is_localized_module S g] : M' ≃ₗ[R] M'' :=
```

Comment:
Is this name consistent with the analogous statement for localization of rings?

Reply:
changed to `is_localized_module.linear_equiv` according to `ring_theory/localization/basic.lean` in [8ea28e2](https://github.com/leanprover-community/mathlib/pull/15559/commits/8ea28e2f7870464e4e4e260cb0a4867d2c1e61dd)